app/models/like.rb

```diff
@@ -1,4 +1,15 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  has_many :posts, dependent: :destroy, foreign_key: :author_id
+  has_many :likes, dependent: :destroy
+  has_many :likes, dependent: :destroy
```

Comment:
- Hey, please, kindly double checkout this as you've made a duplicate. please, fix this to avoid errors in your database.